tests/unittest_nodes.py

```diff
@@ -1583,6 +1583,14 @@ def func():
     """
     )
     assert node.doc == "Docstring"
+    if PY38_PLUS:
+        assert isinstance(node.doc_node, nodes.Const)
+        assert node.doc_node.lineno == 3
+        assert node.doc_node.col_offset == 4
+        assert node.doc_node.end_lineno == 3
+        assert node.doc_node.end_col_offset == 15
+    else:
+        assert node.doc_node is None
```

Comment:
The `if` statement should be moved to only cover some of the line numbering. We want to have the `doc_node` attribute regardless of python version.